docs/book/src/developer/tilt.md

```diff
@@ -192,6 +192,29 @@ COPY --from=tilt-helper /usr/bin/docker /usr/bin/docker
 COPY --from=tilt-helper /go/kubernetes/client/bin/kubectl /usr/bin/kubectl
 ```
 
+
+**extra_args** (Array, default=[]): An array of additional arguments to pass to the main binary configured
+for this provider. Each item in the array will be passed is to the manager. 
+
+Example:
+
+```json
+{
+    "name": "core",
+    "config": {
+        "context": ".",
+        "image": "gcr.io/k8s-staging-cluster-api/cluster-api-controller",
+        "extra_args": ["--feature-gates=MachinePool=true"]
+    }
```

Comment:
Unless I'm mistaken, the example should look more like this as part of tilt-settings.json, it currently looks like it is still formatted for the previous implementation.
```suggestion
    "extra_args": {
        "core": ["--feature-gates=MachinePool=true"],
        "kubeadm-bootstrap": ["--feature-gates=MachinePool=true"],
        "azure": ["--feature-gates=MachinePool=true"]
    }
```